srcStatic/StringConversion.h

```diff
@@ -52,3 +52,10 @@ std::vector<std::string> SplitAndTrim(std::string stringToSplit, char delimiter,
 	}
 	return items;
 }
+
+// Convert hex address value to string
+std::string AddrToHexString(std::size_t addr);
```

Comment:
I wouldn't have thought to put AddrToHexString and GetDateTime in the StringConversion file. However, I think it works fine. 

I like that they are usable by the rest of the code base now.

Reply:
Yes, I've been piling a lot of string methods into there. At some point maybe we should either split it, or rename it to better reflect what it contains.